Tests/id_set.json

```diff
@@ -14270,6 +14270,24 @@
                     "symantec-mss-get-incident"
                 ]
             }
+        }, 
+        {
+            "cveReputation Test": {
+                "name": "cveReputation Test", 
+                "implementing_scripts": [
+                    "VerifyHumanReadableContains", 
+                    "cveReputation"
+                ]
+            }
+        }, 
+        {
+            "cveReputation Test": {
```

Comment:
duplication: re-run the update_id_set.py script

Reply:
this should resolve itself if you commit your code again

Reply:
I believe this is corrected now.